CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# Contributing Guidelines
+
+## Code of Conduct
+
+This project is intended to be a safe, welcoming space for collaboration. All contributors are expected to adhere to the [Contributor Covenant](http://contributor-covenant.org) code of conduct. Thank you for being kind to each other!
+
+## Contributions welcome!
+
+**Before spending lots of time on something, ask for feedback on your idea first!**
+
+Please search [issues](../../issues/) and [pull requests](../../pulls/) before adding something new! This helps avoid duplicating efforts and conversations.
+
+This project welcomes any kind of contribution! Here are a few suggestions:
+
+- **Ideas**: participate in an issue thread or start your own to have your voice heard.
+- **Copy editing**: fix typos, clarify language, and generally improve the quality of the content.
+- **Formatting**: help keep content easy to read with consistent formatting.
+- **Code**: help maintain and improve the project codebase.
+
+
+### Let's Contribute :+1:
+
+- **Step 1** - Fork this repository.
+- **Step 2** - Clone the repository to your local machine.
+- **Step 3** - Resolve the bugs, mentions provided in the Issues section of the repository. _Also add a description of what changes you have done_.
```

Comment:
Can you add the steps for local setup in these steps? It is available on the main README.md file